domains/Tobez.json

```diff
@@ -0,0 +1,11 @@
+ 
```

Comment:
Delete this file please.

Reply:
Not gonna lie, can’t find a way to do it on my phone :/

Reply:
@TheRealTobes28-Codez https://github.com/TheRealTobes28-Codez/register/blob/main/domains/Tobez.json click this link and click the delete button